src/ext/dispatch.h

```diff
@@ -9,12 +9,13 @@
 
 typedef struct _ddtrace_dispatch_t {
     zval callable, function_name;
+    zend_bool append;  // Remove after removing `dd_trace()`
```

Comment:
Bikeshedding comment warning: 
WDYT about calling it something similar  to`postprocess_function_result` -> append looks a bit ambiguous.

Reply:
🤔 I like disambiguating things. 💛 What about `run_as_postprocess`? Or some other suggestion? :)

Reply:
`run_as_postprocess` sounds good!